onnxruntime/python/tools/transformers/fusion_gemmfastgelu.py

```diff
@@ -0,0 +1,77 @@
+# -------------------------------------------------------------------------
```

Comment:
Can you please add docstring, comments and fusion pattern drawings to this file?

A good example to follow could be this one: fusion_attention.py

Reply:
Add it under function fuse.